File: programmers/javascript/level1/K번째수.js
const array = [1, 5, 2, 6, 3, 7, 4];
const commands = [
  [2, 5, 3],
  [4, 4, 1],
  [1, 7, 3],
];

function solution(array, commands) {
  let answer = [];
  for (let i = 0; i < commands.length; i++) {
    let temp = array.slice(commands[i][0] - 1, commands[i][1]);
    temp.sort((a, b) => {
      return a - b;
    });
    answer.push(temp[commands[i][2] - 1]);
  }

  return answer;
}

console.log(solution(array, commands));

File: python/programmers/level1/k번째수.py
def solution(array, commands):
    answer = []
    for com in commands:
        i, j, k = com
        arr = array[i-1:j]
        arr.sort()
        answer.append(arr[k-1])
    return answer


print(solution([1, 5, 2, 6, 3, 7, 4], [[2, 5, 3], [4, 4, 1], [1, 7, 3]]))
